helm/kontroler/deletePods.sh

```
@@ -0,0 +1,24 @@
+    if [ "$FINALIZERS" != "null" ] && [ -n "$FINALIZERS" ]; then
+        echo "Finalizers found: $FINALIZERS"
+
+        # Remove finalizers by patching the pod
+        kubectl patch pod "$POD_NAME" -p '{"metadata":{"finalizers":null}}'
+
+        echo "Finalizers removed from pod $POD_NAME in namespace $NAMESPACE."
+    else
+        echo "No finalizers found for pod $POD_NAME in namespace $NAMESPACE."
+    fi
+
+    kubectl delete pod $POD_NAME
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)